www/templates/build.html

```diff
@@ -57,8 +57,7 @@ <h2 class="head-2">Stop Entire Build Chain</h2>
                 <h2 class="head-2">
                     Resubmit Build
                 </h2>
-                {{ forms.rebuild_build(build_url+"/rebuild"+codebases_arg, authz, sourcestamps[0], builder.name,
-                path_to_builder) }}
+                {{ forms.rebuild_build(build_url+"/rebuild"+codebases_arg, authz, sourcestamps[0], builder.name, path_to_builder, build_chain_top_build_url) }}
```

Comment:
it doesn't seem that this parameter `build_chain_top_build_url ` should be required here